sql_metadb/derived_tables/invoice_adjustments_ext.sql

```diff
@@ -81,4 +81,28 @@ CREATE INDEX ON invoice_adjustments_ext (ratio_of_inv_adj_per_invoice_line);
 
```

Comment:
For some reason, when I ran this query against folio_lotus, I got an error on the derived table `invoice_adjustments_in_addition_to` saying the relation does not exist (line 51). When I specify the `folio_derived` schema for that table, it works fine. I didn't have this trouble when running this query against folio_juniper, but I thought I would mention it.